jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/panel/DynamicAttributesPanel.java

```diff
@@ -202,20 +212,7 @@ protected Component generateFieldComponent(AttributeDefinition attribute) {
                 new ComponentGenerationContext(instanceContainer.getEntityMetaClass(), metaProperty.getName());
         componentContext.setValueSource(valueSource);
 
-        Component resultComponent = uiComponentsGenerator.generate(componentContext);
-
-        setWidth(resultComponent, attribute);
-
-        return resultComponent;
-    }
-
-    protected void setWidth(Component component, AttributeDefinition attribute) {
```

Comment:
i prefer to not configure UI component firstly from "dynattr hot settings". Instead of it it should has behave like normal UI component and configure only from java code or from xml
`else {
            ((HasSize) component).setWidth(fieldWidth);
        }`